src/components/Footer/Footer.js

```diff
@@ -1,5 +1,129 @@
+import FooterTop from './FooterTop';
+import styled from 'styled-components';
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <FooterDiv>
+      <Container>
+        <GlobalUl>
+          {FOOTER_DATA.map(data => {
```

Comment:
```suggestion
          {FOOTER_DATA.map(({id, data, subData}) => {
```
매개변수자리에서 바로 구조분해 할당 진행하면 더 깔끔하겠네요